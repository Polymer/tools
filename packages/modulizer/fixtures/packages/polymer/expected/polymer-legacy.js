/**
@license
Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
Code distributed by Google as part of the polymer project is also
subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
*/
/* template elements */
/* custom-style */
/* bc behaviors */
/* import html-tag to export html */
/*
  FIXME(polymer-modulizer): the above comments were extracted
  from HTML and may be out of place here. Review them and
  then delete this comment!
*/
import { LegacyElementMixin } from './lib/legacy/legacy-element-mixin.js';

import './lib/legacy/polymer-fn.js';
import './lib/legacy/templatizer-behavior.js';
import './lib/elements/dom-bind.js';
import './lib/elements/dom-repeat.js';
import './lib/elements/dom-if.js';
import './lib/elements/array-selector.js';
import './lib/elements/custom-style.js';
import './lib/legacy/mutable-data-behavior.js';
import { html as html$0 } from './lib/utils/html-tag.js';

// bc
export const Base = LegacyElementMixin(HTMLElement).prototype;

// NOTE: this is here for modulizer to export `html` for the module version of this file
export { html$0 as html };
