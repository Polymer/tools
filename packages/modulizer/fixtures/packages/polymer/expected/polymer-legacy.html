<!--
@license
Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
Code distributed by Google as part of the polymer project is also
subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
-->
<script type="module" src="./lib/legacy/legacy-element-mixin.js"></script>
<script type="module" src="./lib/legacy/polymer-fn.js"></script>
<!-- template elements -->
<script type="module" src="./lib/legacy/templatizer-behavior.js"></script>
<script type="module" src="./lib/elements/dom-bind.js"></script>
<script type="module" src="./lib/elements/dom-repeat.js"></script>
<script type="module" src="./lib/elements/dom-if.js"></script>
<script type="module" src="./lib/elements/array-selector.js"></script>
<!-- custom-style -->
<script type="module" src="./lib/elements/custom-style.js"></script>
<!-- bc behaviors -->
<script type="module" src="./lib/legacy/mutable-data-behavior.js"></script>
<!-- import html-tag to export html -->
<script type="module" src="./lib/utils/html-tag.js"></script>
<script type="module">
import './lib/legacy/legacy-element-mixin.js';
import './lib/legacy/polymer-fn.js';
import './lib/legacy/templatizer-behavior.js';
import './lib/elements/dom-bind.js';
import './lib/elements/dom-repeat.js';
import './lib/elements/dom-if.js';
import './lib/elements/array-selector.js';
import './lib/elements/custom-style.js';
import './lib/legacy/mutable-data-behavior.js';
import './lib/utils/html-tag.js';
import { LegacyElementMixin } from '../../../lib/legacy/legacy-element-mixin.js';
import { html as html$0 } from '../../../lib/utils/html-tag.js';

// bc
export const Base = LegacyElementMixin(HTMLElement).prototype;

// NOTE: this is here for modulizer to export `html` for the module version of this file
export { html$0 as html };
</script>
